src/main/kotlin/com/sourcegraph/cody/listeners/CodyDocumentListener.kt

```diff
@@ -28,10 +29,10 @@ class CodyDocumentListener(val project: Project) : BulkAwareDocumentListener {
     }
   }
 
-  override fun documentChangedNonBulk(event: DocumentEvent) {
+  override fun documentChangedNonBulk(event: DocumentEvent) = runInEdt {
```

Comment:
`FileEditorManager.getInstance(project).selectedTextEditor` must be called in EDT - otherwise we can see an internal error